### PR TITLE
feat: add metrics for customization of legend orientation and static legend

### LINKED ISSUE #1434: Static Legend: Measure static legends usage
### Background
We should measure usage around our new features so that we have more information to make future decisions on.

Here Is more about the taxonomy of this as seen in annotations:

> To keep things consistent, we want to organize the metrics into a bit of a taxonomy. The hierarchy should go something like:
> 
> ```
> page -> feature -> ui component -> state
> ```
> 
> For example, on a dashboard page with the annotations ui rendered.
> 
> When a user clicks the toggle to toggle annotating with one click:
> `dashboard.annotations.render_annotation_toggle.clicked`
> 
> Or when a user clicks the graph and saves an annotation using the overlay that pops up
> `dashboard.annotations.add_annotation.added`
> or if they cancel:
> `dashboard.annotations.add_annotation.canceled`
> 

### Acceptance Criteria
- Measurements to include:
  - [x] Static legend toggle is clicked on.
  - [x] What graph type the static legend was toggled on for.

### LINKED ISSUE #1516: Tooltip: measure 'hide hover legend' usage
### Background
We should measure usage around our new features so that we have more information to make future decisions on.

Here Is more about the taxonomy of how this is done:

> To keep things consistent, we want to organize the metrics into a bit of a taxonomy. The hierarchy should go something like:
> 
> ```
> page -> feature -> ui component -> state
> ```
> 
> For example, on a dashboard page with the annotations ui rendered.
> 
> When a user clicks the toggle to toggle annotating with one click:
> `dashboard.annotations.render_annotation_toggle.clicked`
> 
> Or when a user clicks the graph and saves an annotation using the overlay that pops up
> `dashboard.annotations.add_annotation.added`
> or if they cancel:
> `dashboard.annotations.add_annotation.canceled`
> 

### Acceptance Criteria
- Measurements to include:
  - [ ] Hide Hover Legend toggle is clicked on.
  - [ ] What graph type the static legend was toggled on for.

### PR DESCRIPTION
**Pull request by TCL735**
Closes #1434 
Closes #1516

### CODE REVIEW

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 57 | :red_circle:
accessibility | 96 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=1648)

File: src/visualization/components/internal/LegendOrientation.tsx [hunk 1]
```diff
@@ -52,83 +76,77 @@ interface Props extends VisualizationOptionProps {
...
-      })
-    } else {
-      update({
-        legendOpacity: value,
-      })
+const HoverLegendToggle: FC<HoverLegendToggleProps> = ({
+  legendHide,
+  handleSetHoverLegendHide,
+}) => {
+  const getHoverLegendHideStatus = (legendHide: boolean): string => {
```
Comment (Asalem1) [inline src/visualization/components/internal/LegendOrientation.tsx:92]: should this return an `enum` for added stability?

File: src/visualization/components/internal/LegendOrientation.tsx [hunk 2]
```diff
@@ -52,83 +76,77 @@ interface Props extends VisualizationOptionProps {
...
-  const handleSetColorization = (): void => {
-    update({
-      legendColorizeRows: !properties.legendColorizeRows,
+  return (
+    <FlexBox
+      direction={FlexDirection.Row}
+      alignItems={AlignItems.Center}
+      margin={ComponentSize.Medium}
+      stretchToFitWidth={true}
+      style={{marginTop: '0.5em', marginBottom: '1.5em'}}
```
Comment (Asalem1) [inline src/visualization/components/internal/LegendOrientation.tsx:105]: can we set classNames here with some clockface variables as the margin?

File: src/visualization/components/internal/LegendOrientation.tsx [hunk 3]
```diff
@@ -52,83 +76,77 @@ interface Props extends VisualizationOptionProps {
...
-  const percentLegendOpacity = (legendOpacity * 100).toFixed(0)
-
-  const orientationToggle = (
+  return (
     <FlexBox
       direction={FlexDirection.Column}
       margin={ComponentSize.Large}
       alignItems={AlignItems.FlexStart}
-      style={{marginBottom: 18}}
+      style={{marginBottom: '1.5em'}}
```
Comment (Asalem1) [inline src/visualization/components/internal/LegendOrientation.tsx:140]: same as above

**Comment by TCL735** [comment]
Force push was to fix merge conflicts, which were pretty gnarly. Now it is correct with only 2 files changed.

File: src/visualization/components/internal/LegendOrientation.tsx (hunk 2 above)
Comment (TCL735) [reply src/visualization/components/internal/LegendOrientation.tsx]: 👍

**Pull request merged**